tests/ui/or_fun_call.fixed

```diff
@@ -62,7 +62,7 @@ fn or_fun_call() {
     map.entry(42).or_insert_with(String::new);
 
     let mut btree = BTreeMap::<u64, String>::new();
-    btree.entry(42).or_insert_with(String::new);
+    btree.entry(42).or_insert(String::new());
```

Comment:
Hm. This is the second time I saw this unrelated change today. Maybe a rebase required?

Reply:
Yeah... when I rebased sometime ago, that test started failing. Rebased this time and its working.